client/src/theme.ts

```diff
@@ -75,6 +75,8 @@ export const light = {
   errorTitle: colors.gray_60,
   errorBody: colors.gray_50,
   fab: colors.blue_70,
+  toastBackground: colors.blue_70,
```

Comment:
Will we ever need a different colored toast, or is it okay to have single color for all toasts?

Reply:
I think one color would be enough. but I think it's good to be added later.